src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -1640,15 +1668,19 @@ protected function get_attendees_by_order_id( $order_id, $ticket_id = null ) {
 		}
 
 		$find_by_args = [
-			'post_id'   => $order_id,
-			'ticket_id' => [],
+			'post_id'        => $order_id,
+			'ticket_id'      => [],
+			'posts_per_page' => 1,
 		];
 
 		if ( $ticket_id ) {
 			$find_by_args['ticket_id'] = (array) $ticket_id;
 		}
 
-		$orders = Tribe__Tickets__Commerce__PayPal__Order::find_by( $find_by_args );
+		$orders = Tribe__Tickets__Commerce__PayPal__Order::find_by( $find_by_args, [
+			// Get just the paypal ID var we need.
+			'txn_id',
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 5a092157f052d79a82e63aaca95416f4</sub>